consensus/integration/nodes_test.go

```diff
@@ -434,9 +434,11 @@ func createNode(
 
 	// log with node index
 	logConsumer := notifications.NewLogConsumer(log)
-	notifier := pubsub.NewDistributor()
-	notifier.AddConsumer(counterConsumer)
-	notifier.AddConsumer(logConsumer)
+	hotstuffDistributor := pubsub.NewDistributor()
+	hotstuffDistributor.AddConsumer(counterConsumer)
+	hotstuffDistributor.AddConsumer(logConsumer)
+	finalizationDistributor := pubsub.NewFinalizationDistributor()
```

Comment:
We need to pull from master, on master we don't need additional finalization distributor since it's already embedded in [`Notifier`](https://github.com/onflow/flow-go/blob/master/consensus/config.go#L18)